src/encoding.ts

```diff
@@ -67,7 +73,8 @@ export interface Encoding<F> {
   /**
    * Secondary error value of x coordinates for error specified `"errorbar"` and `"errorband"`.
    */
-  xError2?: FieldDefWithoutScale<F> | ValueDef;
+  // `xError2` cannot have type as it should have the same type as `xError`
```

Comment:
`xError` should have the same type as `x`. Therefore `xError` should also be a `SecondaryRangeFieldDef`

Reply:
Please fix in a follow up PR.